aldryn_forms/templatetags/aldryn_forms_tags.py

```diff
@@ -7,6 +7,18 @@
 register = template.Library()
 
 
+@register.assignment_tag(takes_context=True)
+def render_notification_text(context, email_notification, email_type):
+    text_context = context.get('text_context')
+
+    if not (text_context or email_notification):
```

Comment:
I've just spend 1 min trying to figure out what this condition means (don't tell @GaretJax cause he'd laugh at me)... doesn't it mean the following::

     if not text_context and not email_notification:

?